internal/locale/locales/en-us.yaml

```diff
@@ -23,6 +23,8 @@ flag_state_output_description:
   other: "Set the output method. Possible values: plain, simple, json, editor"
 flag_state_non_interactive_description:
   other: Run the State Tool without any prompts
+flag_state_force_description:
+  other: Run the State Tool without any prompts, overriding any safe defaults
```

Comment:
```suggestion
flag_state_non_interactive_description:
  other: Assume default values for any prompts
flag_state_force_description:
  other: Disable prompts and perform the requested action regardless of defaults values (this can result in destructive or unsafe actions, such as installing packages with critical CVEs)
```